### PR TITLE
[iOS/tvOS/MacCatalyst] Disable some of the library tests due to failures

### PR DESCRIPTION
**Pull request by MaximLipnin**
It's to clean up the respective CI staging lanes.

The related issues:
- https://github.com/dotnet/runtime/issues/56073
- https://github.com/dotnet/runtime/issues/56048
- https://github.com/dotnet/runtime/issues/55712
- https://github.com/dotnet/runtime/issues/56084
- https://github.com/dotnet/runtime/issues/56085

### CODE REVIEW

**Comment by karelz** [comment]
Http3_MsQuic test failures are tracked in #56090. Should be fixed and merged shortly. Sorry for the inconvenience!

File: src/libraries/System.Drawing.Common/tests/Drawing2D/HatchBrushTests.cs [hunk 1]
```diff
@@ -54,6 +54,7 @@ public void Ctor_HatchStyle_ForeColor_BackColor(HatchStyle hatchStyle, Color for
         [Theory]
         [InlineData(HatchStyle.Horizontal -1 )]
         [InlineData(HatchStyle.SolidDiamond + 1)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/56048", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
```
Comment (akoeplinger) [inline src/libraries/System.Drawing.Common/tests/Drawing2D/HatchBrushTests.cs:57]: @MaximLipnin you can use `[ConditionalTheory(Helpers.IsDrawingSupported)]` instead, that will disable the test on platforms where System.Drawing.Common is not supported

File: src/libraries/System.Drawing.Common/tests/ImageTests.cs [hunk 2]
```diff
@@ -520,6 +520,7 @@ public void FromFile_InvalidBytes_ThrowsOutOfMemoryException(byte[] bytes)
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/56048", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
```
Comment (akoeplinger) [inline src/libraries/System.Drawing.Common/tests/ImageTests.cs:523]: same here, use `[ConditionalFact(Helpers.IsDrawingSupported)]` instead

**Review by akoeplinger — APPROVED** [review]

File: src/libraries/System.Drawing.Common/tests/Drawing2D/HatchBrushTests.cs (hunk 1 above)
Comment (MaximLipnin) [reply src/libraries/System.Drawing.Common/tests/Drawing2D/HatchBrushTests.cs:57]: @akoeplinger Is it for all the System.Drawing.Common tests which I've disabled or just for the two you mentioned?

Comment (akoeplinger) [reply src/libraries/System.Drawing.Common/tests/Drawing2D/HatchBrushTests.cs:57]: for all

Comment (MaximLipnin) [reply src/libraries/System.Drawing.Common/tests/Drawing2D/HatchBrushTests.cs]: done, thanks

File: src/libraries/System.Drawing.Common/tests/ImageTests.cs (hunk 2 above)
Comment (MaximLipnin) [reply src/libraries/System.Drawing.Common/tests/ImageTests.cs]: done, thanks

File: src/libraries/System.Drawing.Common/tests/Drawing2D/HatchBrushTests.cs (hunk 1 above)
Comment (MaximLipnin) [reply src/libraries/System.Drawing.Common/tests/Drawing2D/HatchBrushTests.cs]: weird, the System.Drawing.Common tests keep failing one per run. Perhaps, it makes sense to use ` [ConditionalClass(Helpers.IsDrawingSupported)]`

**Comment by MaximLipnin** [comment]
@steveisok Could you take a look please?

**Review by steveisok — APPROVED** [review]

**Pull request merged**